tests/layer_manager.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { AnyModel } from '@anywidget/types';
+import '../js/layer_manager';
```

Comment:
Nit: Blank line to separate external packages from internal imports.

Reply:
Done.